evaluation/data/ims/sorc/regrid_ims_to_gaussian.f90

```diff
@@ -0,0 +1,418 @@
+program regrid_ims_cgrid
+
+use netcdf
+
+implicit none
+
+character*100 :: map_filepath
+character*100 :: ims_filepath
+character*100 :: regrid_outpath
+
+character*256 :: ims_source_path
+character*256 :: ims_out_path
+
+character*256 :: land_fix_filename
+character*256 :: ims_filename
+character*256 :: map_filename 
+character*256 :: output_filename 
+
+character*10   :: obs_source
+character*10   :: fv3_grid
+character*3    :: ims_res
+
+character*7   :: datestring
+double precision  :: sec_since
+integer           :: offset_ss
+integer           :: dim_time, id_time, id_lat, id_lon
+integer           :: dim_id_j, dim_id_i
+integer           :: i, j, itotal, tile_i, tile_j 
+character*19      :: current_date  ! current date
+character*19      :: since_date = "1970-01-01 00:00:00"
+character*8       :: daystring
+character*4       :: yyyy
+character*2       :: mm, dd
+
+integer*2, allocatable :: imssnow_flag    (:,:)
+integer*2, allocatable :: lookup_i        (:,:)
+integer*2, allocatable :: lookup_j        (:,:)
+
+real     , allocatable :: latitude(:)
+real     , allocatable :: longitude(:)
+real     , allocatable :: xcb(:)  ! destination lon
```

Comment:
xcb should be more descriptive name

Reply:
Just leave current name now as it is actual variable in corner file